exercises/practice/pangram/pangram.vader

```diff
@@ -1,36 +1,34 @@
-"
-" Version: 1.1.0
-"
-
-Before:
-  unlet! input expected
-
-Execute (sentence empty):
+Execute (empty sentence):
```

Comment:
Unless this is coming from upstream, perhaps this should be phrase, since not all pangrams are sentences.  (As evidenced by this empty "sentence".  While it is probably accurate to describe this as an empty string.

Reply:
I agree, but that's the description upstream unfortunately. However, we have some leeway here as long as it's clear that "empty string" here is the "empty sentence" from the tests.toml. I think it's safe to make that tweak.